packages/backend/src/lib/backends/base.sync.ts

```diff
@@ -86,7 +86,7 @@ export abstract class BaseSync {
         refresh
       });
 
-      logger.trace(JSON.stringify(result));
+      //logger.trace(JSON.stringify(result));
```

Comment:
Should this be deleted? or we just want to leave it there for testing?

Reply:
I figured it might be useful for debugging later so left it in. 🤷